simulator/modules/clients.py

```diff
@@ -0,0 +1,79 @@
+from utils import get_logger
+import simpy
+
+class Request(object):
+
+    def __init__(self, created_at, client, load_balancer, conf, log_path):
+        self.created_at = created_at
+        self.client = client
+        self.load_balancer = load_balancer
+
+        self.service_time = float(conf['service_time'])
+        self.memory = float(conf['memory'])
+
+        self.done = False
+        self._sent_time = None
+        self._arrived_time = None
+        self._finished_time = None
+        self._latence_time = None
+
+        self.logger = get_logger(log_path + "/request.log", "REQUEST")
+
+    def run(self, env, heap):
+        yield env.timeout(self.service_time)
+        yield heap.put(self.memory)
+
+    def sent_at(self, time):
+        self._sent_time = time
+
+    def arrived_at(self, time):
+        self._arrived_time = time
+
+    def finished_at(self, time):
+        self._finished_time = time
+        self._latence_time = self._finished_time - self._sent_time
+        self.logger.info(" At %.3f, Request was finished. Latence: %.3f" % (self._finished_time, self._latence_time))
```

Comment:
Wouldn't be much better to log in a format where we can process later (i.e. CSV)?

Reply:
We have added the logging logic to the simulator aiming at make easier the debugging job. 
I'll ensure generate the results with that format. Thanks for suggest!

Reply:
I believe CSV is super simple and cleaner (much easier to debug). But it was only a suggestion, won't block the PR.